prober/prober.go

```diff
@@ -26,6 +25,15 @@ type Prober struct {
 // ProberOption is a function modifying some parameters of the Prober
 type ProberOption func(p *Prober)
 
+// NotFoundError
+type NotFoundError struct {
+	Err string
+}
+
+func (nferr NotFoundError) Error() string {
+	return nferr.Err
+}
```

Comment:
nitpick: we usually name the error variable `err`. I don't see a good reason to change this convention here
```suggestion
func (err NotFoundError) Error() string {
  return fmt.Sprintf("probe %v is not present in prober", err.ProbeName)
}
```